### ISSUE TITLE
SR-18-0000212 FD Elevator Inspection Status

### ISSUE THREAD

**Issue by RikiSjachrani-COM** [City-of-Madison/Development-Team-2#44]
FD Elevator Inspection Status - 
Specifications
Tool:
Inspection Status - Add "Corrected, violations remaining" to the reinspection result group
EMSE Scripting - Above should behave like corrected

**Comment by RikiSjachrani-COM** [comment]
@JaneSchneider-COM we met with Jerry on this right? is there anything we are waiting for with this

**Comment by JaneSchneider-COM** [comment]
@RikiSjachrani-COM : Yes, we had a phone meeting w/Jerry. I don't think we are waiting on anything specifically from them at this point. I may have questions later once I did into this SR.

**Comment by RikiSjachrani-COM** [comment]
@JaneSchneider-COM could we re-assign this to someone while you are out next week? @JacquelineGoltz-COM we could loop in Jim or Alejandro?

**Comment by JaneSchneider-COM** [comment]
@RikiSjachrani-COM: Regarding reassigning: Sure.

**Comment by JacquelineGoltz-COM** [comment]
@RikiSjachrani-COM is this going to be reassigned to Alejandro or is Jane to continue?

**Comment by RikiSjachrani-COM** [comment]
Whoever can get to it...it's lower priority. It would be nice to get it over with...
